pallets/ctype/src/benchmarking.rs

```diff
@@ -16,26 +16,40 @@
 
 // If you feel like getting in touch with us, you can do so at info@botlabs.org
 
-use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite};
-use frame_system::RawOrigin;
+use sp_std::{
+	convert::{TryFrom, TryInto},
+	vec::Vec,
+};
 
 use crate::*;
+use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite};
+use frame_support::traits::{Currency, Get};
+use frame_system::RawOrigin;
 
 const SEED: u32 = 0;
 
 benchmarks! {
+	where_clause {
+		where
+		<<T as Config>::Currency as Currency<AccountIdOf<T>>>::Balance: TryFrom<usize>,
+		<<<T as pallet::Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::Balance as TryFrom<usize>>::Error: sp_std::fmt::Debug,
```

Comment:
I think the two lines are almost the same with the exception of the error at the end, right? Then we should use either `T as Config` or `T as pallet::Config` and also either `Currency<<T as frame_system::Config>::AccountId>` or `Currency<AccountIdOf<T>>`. I would vote for `T as Config` and `Currency<AccountIdOf<T>>`. Plus, why do we need the error to implement Debug?

Reply:
7cee163